enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/Søknadsfelt.kt

```diff
@@ -1,3 +1,5 @@
 package no.nav.familie.kontrakter.ef.søknad
 
-data class Søknadsfelt<T>(val label: String, val verdi: T, val alternativer: List<String>? = null)
+data class Søknadsfelt<T>(val label: String, val verdi: T, val alternativer: List<String>? = null, val svarId: T? = null)
```

Comment:
Kan svarId være noe annet enn String? Eller hvordan virker lister?

Reply:
Kan være ```List<String>``` for noen felt.

Reply:
De feltene som er av typen boolean da. Hvordan er de tenkt å håndteres?
Fra frontend kommer de inn som eks 
```
skalGifteSegEllerBliSamboer: {
label: "Har du konkrete planer om å gifte deg eller bli samboer?"
spørsmålid: "skalGifteSegEllerBliSamboer"
svarid: "NEI"
verdi: false
}
```

Der er svarId en string, men verdi er boolean

Reply:
Til informasjon til andre (diskuterte med Johan på slack): Vi trenger ikke svarId fra ```Søknadsfelt<Boolean>```. Men vurderer å sende det med likevel fra ef-soknad-api.